BUILDING.md

```diff
@@ -27,22 +27,14 @@ On OS X, you will also need:
 On FreeBSD and OpenBSD, you may also need:
 * libexecinfo
 
-To build Node.js:
+In order to build Node.js, the first listing of `python` in your `PATH`
+environment variable must point to the Python 2.6/2.7 binary. To build:
```

Comment:
I don't think this text should replace `To build Node.js:`.

I'd rather see something like:

> To build Node.js:
> ```console
> $ ./configure
> $ make
> ```
> 
> Note that the above requires that  `python` resolve to Python 2.6/2.7 and not a newer version.